atom/browser/api/atom_api_window.cc

```diff
@@ -786,6 +786,18 @@ bool Window::IsVisibleOnAllWorkspaces() {
   return window_->IsVisibleOnAllWorkspaces();
 }
 
+void Window::SetVibrancy(v8::Local<v8::Value> value, mate::Arguments* args) {
+  std::string type;
```

Comment:
If this just took an `args` param then you could support `null` and `undefined` with:

```js
std::string type;
args->GetNext(&type);
window_->SetVibrancy(type);
```

`type` will be empty if the first argument is missing or is not a string.